From: Matthew Wilcox <matthew.r.wilcox@intel.com>
Date: Thu, 20 Jan 2011 09:14:34 -0500
Subject: NVMe: Reduce set_queue_count arguments by one
Git-commit: b3b06812e199f248561ce7824a4a8a9cd573c05a
References: FATE#313627
Patch-Mainline: v3.7-rc1

sq_count and cq_count are always the same, so just call it 'count'.

Signed-off-by: Matthew Wilcox <matthew.r.wilcox@intel.com>
Acked-by: Hannes Reinecke <hare@suse.de>
---
 drivers/block/nvme.c |    6 +++---
 1 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/block/nvme.c b/drivers/block/nvme.c
index b10e064..7efd7e9 100644
--- a/drivers/block/nvme.c
+++ b/drivers/block/nvme.c
@@ -777,12 +777,12 @@ static void nvme_ns_free(struct nvme_ns *ns)
 	kfree(ns);
 }
 
-static int set_queue_count(struct nvme_dev *dev, int sq_count, int cq_count)
+static int set_queue_count(struct nvme_dev *dev, int count)
 {
 	int status;
 	u32 result;
 	struct nvme_command c;
-	u32 q_count = (sq_count - 1) | ((cq_count - 1) << 16);
+	u32 q_count = (count - 1) | ((count - 1) << 16);
 
 	memset(&c, 0, sizeof(c));
 	c.features.opcode = nvme_admin_get_features;
@@ -800,7 +800,7 @@ static int __devinit nvme_setup_io_queues(struct nvme_dev *dev)
 {
 	int this_cpu;
 
-	set_queue_count(dev, 1, 1);
+	set_queue_count(dev, 1);
 
 	this_cpu = get_cpu();
 	dev->queues[1] = nvme_create_queue(dev, 1, NVME_Q_DEPTH, this_cpu);
-- 
1.7.4.2

